### ISSUE TITLE
Aurora crystals

### ISSUE THREAD

**Issue by aqpoyo** [NillerMedDild/Enigmatica6#2011]
Well this is not a suggestion but I don't know where to put it. So I have been looking online where the aurora crystals generate in the end so I can craft crystal shards but I couldn't find it. I have been looking for quite a long time around the end but couldn't find them either. Are the disabled? Also the amber ore does not generate in the end? I made some with the orechid but I haven't found them in the end.

Thanks!

**Comment by aqpoyo** [comment]
Ok sorry for asking trought here. Thanks for the tip but the problem was that the better end forge biome where not generating. I look in a copy world with the locate biome and didn't found anything. So what I did is make another copy of the world, I deleted the end chunks I already visited and then I changed the biome source for the end dimension inside your world level.dat to: betterendforge:betterend_biome_provider like they say here [https://www.curseforge.com/minecraft/mc-mods/betterend-forge-port](url). After that it worked good and the biomes are generating and found both of them.

It was probably because my world was generated in 0.3.x version of the pack no?

Btw, if anyone on a Mac need to update the level.dat, they can use this online tool [http://irath96.github.io/webNBT/](url), I couldn't download the nbtexplorer.
